tests/commerce_integration/TEC/Tickets/Commerce/Partials/Gateway/PayPal/__snapshots__/AdvancedPayments__test_should_render__0.snapshot.html

```diff
@@ -1,6 +1,6 @@
 
 <!-- Advanced credit and debit card payments form -->
-<div class="tribe-tickets__commerce-checkout-paypal-advanced-payments-container">
+<div class="tribe-tickets__commerce-checkout-paypal-advanced-payments-container tribe-common-a11y-hidden">
```

Comment:
We need to add the `@since TBD` and update the `@version TBD` for templates whenever we make changes.